pages/phases/phone-assessment-interviews/training.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: article
+permalink: /hiring-phases/phone-assessment-interviews/training/
+section: hiring-phases
+sidenav: hiring-phases
+phase: phone-assessment-interviews
+sub-phase-order: 011
+title: Training SMEs to Conduct Interviews
+intro: While phone interviews are being scheduled with applicants, facilitate a mandatory two-hour interview training for all SMEs who will conduct the next round of interviews that explains how to conduct structured interviews and assess an applicant’s proficiency with the required competencies based on their responses.
+---
+
+### Setting Up Phone Assessment Interview Training
+
+Send calendar invites to all SMEs who will conduct the next round of interviews for your hiring action. SMEs can participate in the phone interview training in person or remotely using a conference bridge line, depending on the means and norms at your agency.
+
+To prepare for your training:
+- Update the `[Phone Interview Training Presentation Template](toolkit-link)` with your agency-specific information, including both the questions for that interview round and final competencies and proficiencies from your job analysis workshop.  
```

Comment:
You can find the toolkit links for this phase here:
https://github.com/labopm/competitive-hiring-pilot/blob/master/_includes/toolkit/phone-assessment-interviews.html

Just copy the link out and drop the class (the class is specific to when that link is in the toolkit itself).

Reply:
As far as I can tell, the actual PPT template isn't on the site yet, is it?

Reply:
Touche. It is not. Noticed Martha just posted it to Slack. So it will be there soon!